### PR TITLE
[10.0][FIX] account_asset: fix performance generating many posted moves

### PR DESCRIPTION
**Pull request by luismontalba**
Description of the issue/feature this PR addresses:
Improve/fix performance when generating a big number (e.g. 1000) of posted and not grouped moves, by the automatic option.

Current behavior before PR:
Process can't finish.

Desired behavior after PR is merged:
Process finish.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

@Tecnativa

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account_asset/models/account_asset.py [hunk 1]
```diff
@@ -406,7 +406,9 @@ def _compute_entries(self, date, group_entries=False):
             ('move_check', '=', False)])
         if group_entries:
             return depreciation_ids.create_grouped_move()
-        return depreciation_ids.create_move()
+        for depreciation in depreciation_ids:
+            created_moves += depreciation.create_move()
```
Comment (pedrobaeza) [inline addons/account_asset/models/account_asset.py]: It lacks the definition of the variable: `created_moves = []`

**Comment by pedrobaeza** [comment]
The CLA is still incorrect

**Comment by luismontalba** [comment]
Changes done. I'm trying to solve de CLA issue

**Comment by pedrobaeza** [comment]
@luismontalba please rebase your branch over current 10.0 branch to see if this fixes runbot. About the CLA, do you have the corporate email (luis.montalba@tecnativa.com) in the GitHub account (https://github.com/luismontalba) and it's visible for the CLA bot to be checked?

**Comment by luismontalba** [comment]
Yes, I have my corporate email as public but it is luis.martinez@tecnativa.com

imagen

**Comment by pedrobaeza** [comment]
Agh, that's the problem. The CLA is with an incorrect mail. I will make a PR fixing it.

**Comment by pedrobaeza** [comment]
Closing as no more interest in this version.

**Pull request closed without merging**